src/api/post/index.ts

```diff
@@ -1,7 +1,7 @@
 import http from '@/api/core';
-import { CreatePost, SubTravelogue } from '@/types/post';
+import { CreatePostType, SubTravelogue } from '@/types/post';
```

Comment:
p1;

```suggestion
import { CreatePostType, SubTravelogueType } from '@/types/post';
```

혜준님 코드 리뷰하면서 새롭게 추간된 Type 컨벤션입니다. PR 머지 전에 반영해주세요.

[추가된 Type 컨벤션](https://github.com/prgrms-web-devcourse/Team-Moongsal-Travel.zip-FE/pull/63#discussion_r1125425422)

Reply:
혜준님이 작업한 부분이라서 혜준님이 변경해야겠네요. @solar3070